specification/billing/resource-manager/Microsoft.Billing/preview/2018-08-01-preview/examples/BillingAccountInvoicesList.json

```diff
@@ -0,0 +1,63 @@
+{
+  "parameters": {
+    "api-version": "2018-06-30",
+    "billingAccountId": "2000200",
+    "periodStartDate": "2018-01-01",
+    "periodEndDate": "2018-06-30",
+    "$filter": "invoicePeriodStartDate le 2018-06-01"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "providers/Microsoft.Billing/billingAccounts/2000200/billingProfile/345986/invoices/1383723",
+            "name": "1383723",
+            "type": "Microsoft.Billing/invoices",
+            "properties": {
+              "date": "2018-01-16",
+              "status": "Due",
+              "amountDue": 2000.00,
+              "billedAmount": 2000.00,
+              "currency": "USD",
+              "invoicePeriodStartDate": "2018-01-01",
+              "invoicePeriodEndDate": "2018-01-15",
+              "billingProfile": "345986",
+              "billingProfileName": "Contoso Operations Billing",
+              "purchaseOrderNumber": "123456",
+              "documentUrls": [
+                {
+                  "kind": "Invoice",
+                  "url": "https://microsoft.com/invoice.pdf"
+                }
+              ]
+            }
+          },
+          {
```

Comment:
Additional properties not allowed:
    documentUrls,purchaseOrderNumber,billingProfile,currency,billedAmount,amountDue,status,date